src/test/java/api/loans/DueDateScheduledNoticesTests.java

```diff
@@ -280,10 +283,11 @@ public void noticesShouldBeRescheduledAfterRenewal()
         uponAtDueDateNoticeConfiguration,
         afterDueDateNoticeConfiguration));
 
-    useLoanPolicyAsFallback(
+    useFallbackPolicies(
```

Comment:
Can you replace this by `use(noticePolicy)`?

Reply:
Done.